### PR TITLE
OPSEXP-2627: bump components for Fornax release

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-2627

Tests of Deprecated versions: https://github.com/Alfresco/alfresco-ansible-deployment/actions/runs/8733936648

### CODE REVIEW

**Comment by alxgomz** [comment]
> We won't be updating test configs for 7.1 and 7.2?

Well spotted! I missed it since we removed the deprecated versions from the automated CI. Wil l run the job for deprecated versions as it's a release merge

**Review by pmacius — APPROVED** [review]

File: group_vars/all.yml [hunk 1]
```diff
@@ -181,11 +181,11 @@ use_custom_keystores: false
 # to default ansible merging behaviour
 dependencies_version:
   postgresql_connector: 42.6.1
-  postgres_major_version: 15
```
Comment (gionn) [inline group_vars/all.yml:184]: the last release we did included 15, if now we are reverting this is a breaking change for existing environments

I am not sure how but if it's required only for centos 7 then only centos 7 should stick to 14

**Review by gionn — CHANGES_REQUESTED** [review]

File: group_vars/all.yml (hunk 1 above)
Comment (gionn) [reply group_vars/all.yml:184]: the only less ugly thing I can think of, is to add an assert task in the pre-checks playbook that if using postgres and current os is centos 7 and current postgres > 14 then they should downgrade

Comment (alxgomz) [reply group_vars/all.yml:184]: I'm not sure what use case you are trying to cover for. If users specify a postgres host then it should not have any postgres installation on it - neither from a manual installation, nor from a a run of a previous version of the playbook - as that's not something we support

**Review by gionn — APPROVED** [review]

**Pull request merged**